### DISCUSSION TITLE
Nussknacker streaming-lite troubleshooting guide

### DISCUSSION THREAD

**Discussion by christian-edelsbrunner** [TouK/nussknacker#2986] [Q&A]
Hi! 

Is there a trouble shooting guide for nussknacker light? 

So in my example: I see in the consumer groups that messages get consumed by the deployed scenario. But somehow they get lost somewhere and do not land in the target topic. 

I checked the logs of the deployed scenario (pods that got deployed fr that scenario) as well as the metrics. But no clue why those messages are "lost" somewhere. I assume it is some wrong SpEL but no error message or exception anywhere. 

Is there any guideline or docu which would help me to dig into that? (like where can/should I increase the log levels ....)

**Comment by arkadius** [comment]
Hi,
Can you share with us exported scenario? Does it has some filters? Do you see some non-zero values in Rejected events graph chart in grafana? What about errors and Source throughput chart?

As regards to logging level, if you want to increase log level of runtime containers, you can do it by customizing k8s deployments via [k8sDeploymentConfig](https://nussknacker.io/documentation/docs/installation_configuration_guide/DeploymentManagerConfiguration#customizing-k8s-deployment) and set NU_LOG_LEVEL environment variable to DEBUG (default is INFO). I assume that you are using docker image in version 1.2 (in next version this environment variable will be renamed to NUSSKNACKER_LOG_LEVEL). After this change, you have to restart designer and redeploy scenario. We have plans to make it easier by exposing logback configuration in configmap. 

Also have you checked logs of all runtime containers (in designer container you rather won't see anything interesting).

**Comment by dswiecki** [comment]
Hi @an0r0c 
You can find some operations related configuration and troubleshooting docs here [lite operations](https://nussknacker.io/documentation/docs/operations_guide/Lite#logging-level)
